dandi/dandiapi.py

```diff
@@ -132,15 +132,20 @@ def send_request(
             _headers["accept"] = "application/json"
 
         lgr.debug("%s %s", method.upper(), url)
-        result = f(
-            url,
-            params=parameters,
-            data=data,
-            files=files,
-            json=json,
-            headers=_headers,
-            **kwargs,
-        )
+        try:
+            result = f(
+                url,
+                params=parameters,
+                data=data,
+                files=files,
+                json=json,
+                headers=_headers,
+                **kwargs,
+            )
+        except Exception:
+            lgr.exception("HTTP connection failed")
```

Comment:
```suggestion
            lgr.exception("HTTP connection failed: %s", exc)
```
so we log how it failed

Reply:
[The `logging.exception()` method](https://docs.python.org/3/library/logging.html#logging.Logger.exception) already adds exception information.

Reply:
oh, never used it. Good to know, thanks!